dashboardsly/static/css/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignore everything except this gitignore
+*
+!.gitignore
```

Comment:
:cow2: Pretty normal pattern, except I'd put a comment about why you're doing this in the .gitignore itself.